portfolio/pom.xml

```diff
@@ -38,6 +44,14 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+
+    
+  
+    <dependency>
```

Comment:
Looks like you have this gson dependency included twice. Probably delete this one and keep the one above.